Closure_Front_End/src/views/Home.vue

```diff
@@ -158,9 +159,12 @@ export default {
   created() {
     if (this.$auth.isAuthenticated.value) {
       this.$auth.getIdTokenClaims().then(console.log, console.error);
-      this.$http.get("tracks/?track_number=3010").then(response => {
+      this.$http.get("tracks/?track_number=3010",
+      {validateStatus: Utils.validateStatusCode(status)}
+      ).then(response => {
         this.track = response.data.results[0]
       })
+      .catch (Utils.popOutStatusCodeError);
```

Comment:
this isn't my change:
https://github.com/ScDor/Closure/blob/master/Closure_Front_End/src/views/Home.vue#:~:text=%7D%2C-,created()%20%7B,%7D,-%7D%2C
this is the default home page (main page) I guess that @noabarlia added it with default values since it is easier to develop with the hardcoded values (there is more default hardcoded values above). It will need to be removed before the site is aired, but for now, it is ok. @noabarlia correct me if I'm wrong.

Reply:
Thanks, opened #93

Reply:
Yes, it's there for development purposes